include/UIKit/UIImage.h

```diff
@@ -36,95 +40,130 @@
 #import <QuartzCore/CALayer.h>
 
 enum {
-    UIImageOrientationUp,
-    UIImageOrientationDown, // 180 deg rotation
-    UIImageOrientationLeft, // 90 deg CCW
-    UIImageOrientationRight, // 90 deg CW
-    UIImageOrientationUpMirrored, // as above but image mirrored along
-    // other axis. horizontal flip
-    UIImageOrientationDownMirrored, // horizontal flip
-    UIImageOrientationLeftMirrored, // vertical flip
-    UIImageOrientationRightMirrored, // vertical flip
+  UIImageOrientationUp,
+  UIImageOrientationDown,       // 180 deg rotation
+  UIImageOrientationLeft,       // 90 deg CCW
+  UIImageOrientationRight,      // 90 deg CW
+  UIImageOrientationUpMirrored, // as above but image mirrored along
+  // other axis. horizontal flip
+  UIImageOrientationDownMirrored,  // horizontal flip
+  UIImageOrientationLeftMirrored,  // vertical flip
+  UIImageOrientationRightMirrored, // vertical flip
 };
 typedef uint32_t UIImageOrientation;
 
 enum {
-    UIImageResizingModeTile,
-    UIImageResizingModeStretch,
+  UIImageResizingModeTile,
+  UIImageResizingModeStretch,
 };
 typedef uint32_t UIImageResizingMode;
 
-enum { UIImageRenderingModeAutomatic, UIImageRenderingModeAlwaysOriginal, UIImageRenderingModeAlwaysTemplate };
+enum {
+  UIImageRenderingModeAutomatic,
+  UIImageRenderingModeAlwaysOriginal,
+  UIImageRenderingModeAlwaysTemplate
+};
 typedef int32_t UIImageRenderingMode;
 
 @class CIImage, UIImageAsset, UITraitCollection;
 
 UIKIT_EXPORT_CLASS
 @interface UIImage : NSObject <NSSecureCoding>
 
-@property (nonatomic, readonly) CGFloat scale;
-@property (nonatomic, readonly) CGImageRef CGImage;
-@property (nonatomic, readonly) CGSize size;
-@property (nonatomic, readonly) NSArray* images STUB_PROPERTY;
-@property (nonatomic, readonly) NSInteger leftCapWidth;
-@property (nonatomic, readonly) NSInteger topCapHeight;
-@property (nonatomic, readonly) NSTimeInterval duration;
-@property (nonatomic, readonly) UIImageRenderingMode renderingMode;
-@property (readonly, copy, nonatomic) UITraitCollection* traitCollection STUB_PROPERTY;
-@property (readonly, nonatomic) BOOL flipsForRightToLeftLayoutDirection STUB_PROPERTY;
-@property (readonly, nonatomic) CIImage* CIImage STUB_PROPERTY;
-@property (readonly, nonatomic) UIEdgeInsets alignmentRectInsets STUB_PROPERTY;
-@property (readonly, nonatomic) UIEdgeInsets capInsets;
-@property (readonly, nonatomic) UIImageAsset* imageAsset STUB_PROPERTY;
-@property (readonly, nonatomic) UIImageOrientation imageOrientation;
-@property (readonly, nonatomic) UIImageResizingMode resizingMode STUB_PROPERTY;
+@property(nonatomic, readonly) CGFloat scale;
+@property(nonatomic, readonly) CGImageRef CGImage;
```

Comment:
Hopefully so :L)

---
In reply to: [84150620](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84150620) [](ancestors = 84150620)